integration_tests/expected_data/sars_cov_2/ont_unknown/results.csv

```diff
@@ -1,7 +1,7 @@
 SAMPLE_ID,STATUS,CONTAMINATED_READS,PRIMER_DETECTED,PRIMER_NUMREADS,PRIMER_UNIQUE_NUMREADS,PRIMER_AMBIGUOUS_NUMREADS,PRIMER_COVERAGE,PRIMER_INPUT,NCOV_PCT_N_BASES,NCOV_PCT_COVERED_BASES,NCOV_LONGEST_NO_N_RUN,NCOV_NUM_ALIGNED_READS,NCOV_QC_PASS,PANGOLIN_LINEAGE,PANGOLIN_CONFLICT,PANGOLIN_AMBIGUITY_SCORE,PANGOLIN_SCORPIO_CALL,PANGOLIN_SCORPIO_SUPPORT,PANGOLIN_SCORPIO_CONFLICT,PANGOLIN_SCORPIO_NOTES,PANGOLIN_PANGO_DESIGNATION_VERSION,PANGOLIN_VERSION,PANGOLIN_SCORPIO_VERSION,PANGOLIN_CONSTELLATION_VERSION,PANGOLIN_IS_DESIGNATED,PANGOLIN_QC_STATUS,PANGOLIN_QC_NOTES,PANGOLIN_NOTE
-2cf78ed4-edae-4943-ab04-5d7a1e4cbdcb,Completed,7,none,0,0,0,0.0,unknown,12.35,87.71,3925,33005,True,B.1.1.7,0.0,,Alpha (B.1.1.7-like),0.74,0.04,scorpio call: Alt alleles 17; Ref alleles 1; Amb alleles 5; Oth alleles 0,PUSHER-v1.18.1.1,4.2,0.3.17,v0.1.10,False,pass,Ambiguous_content:0.14,Usher placements: B.1.1.7(1/1)
-980f552f-9e50-4d0a-a9fc-dc1398a8bebf,Completed,176,NEB-VarSkip_v1a-long,53324,25,0,1.0,unknown,29.97,69.77,1241,9317,True,BA.1.1,0.0,,Probable Omicron (BA.1-like),0.66,0.0,scorpio call: Alt alleles 39; Ref alleles 0; Amb alleles 16; Oth alleles 4,PUSHER-v1.18.1.1,4.2,0.3.17,v0.1.10,False,pass,Ambiguous_content:0.3,Usher placements: BA.1.1(25/68) BA.1.1.12(9/68) BA.1.1.13(9/68) BA.1.1.14(12/68) BA.1.1.2(13/68)
-9cae232f-6fc6-4a1c-ad06-99ca23b01bfa,Completed,0,none,0,0,0,0.0,unknown,1.19,98.71,25082,35394,True,B.1,0.0,,,,,,PUSHER-v1.18.1.1,4.2,0.3.17,v0.1.10,False,pass,Ambiguous_content:0.02,Usher placements: B.1(1/1)
-b833399a-4d49-4d00-abdb-39d5086a5a0d,Completed,1,none,0,0,0,0.0,unknown,14.22,85.9,3647,25186,True,B.1.1.7,0.0555555555555555,,Alpha (B.1.1.7-like),0.74,0.04,scorpio call: Alt alleles 17; Ref alleles 1; Amb alleles 5; Oth alleles 0,PUSHER-v1.18.1.1,4.2,0.3.17,v0.1.10,False,pass,Ambiguous_content:0.15,Usher placements: B.1.1.7(17/18) Q.1(1/18)
-d3e958b5-0c56-48d6-807b-6a89f61408ce,Completed,4,none,0,0,0,0.0,unknown,23.08,76.55,2201,19057,True,B.1.1.7,0.0,,,,,,PUSHER-v1.18.1.1,4.2,0.3.17,v0.1.10,False,pass,Ambiguous_content:0.24,Usher placements: B.1.1.7(12/12)
-d95c7267-c0b0-4b53-b1ba-91fb85cc109e,Completed,0,none,0,0,0,0.0,unknown,9.78,90.63,4461,8300,True,P.1,0.0,,Gamma (P.1-like),0.81,0.0,scorpio call: Alt alleles 13; Ref alleles 0; Amb alleles 1; Oth alleles 2,PUSHER-v1.18.1.1,4.2,0.3.17,v0.1.10,False,pass,Ambiguous_content:0.11,Usher placements: P.1(2/3) P.1.12(1/3)
+2cf78ed4-edae-4943-ab04-5d7a1e4cbdcb,Completed,7,none,0,0,0,0.0,unknown,43.47,56.23,2797,18498,True,B.1.1.7,,,Alpha (B.1.1.7-like),0.7,0.04,scorpio call: Alt alleles 16; Ref alleles 1; Amb alleles 6; Oth alleles 0,SCORPIO_v0.1.10,4.2,0.3.17,v0.1.10,False,fail,Ambiguous_content:0.45,scorpio called lineage B.1.1.7
```

Comment:
For some of ONT integration test samples, primer autodetection does not find any primer within our set of supported primers. By default, ncov, which is executed downstream of primer autodetection, requires a primer as input parameter. If the `input_primer` is `unknown` and `primer_detected` is `none`, by default the latest artic primer is used.
Before this PR, the default primer was ARTIC V4.1. Now it is ARTIC V5.3.2-400.

Therefore, these samples with no detected primer report slightly different data because of this default primer. Personally, I would not expect this to happen, but it looks like that ARTIC minion is actually doing some trimming, hence the different output metrics. This could also be explained by internal issues within ARTIC minion. The new primer is supposed to have higher quality, so, from this perspective, I could imagine that the new results are somehow more accurate. Interestingly, looking at the new results, it seems to me that the output is kind of more cautious. A few samples now fail pangolin QC, suggesting that there is little confidence in assigning a lineage.

Reply:
Investigated: https://jira.congenica.net/browse/PSG-3152